<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ConnectionList" name="CL" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutrojan_go_desktop">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="ActionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="ActionAdd"/>
    <addaction name="ActionEdit"/>
    <addaction name="ActionShare"/>
    <addaction name="ActionDel"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="ActionConnect"/>
    <addaction name="ActionDisconnect"/>
    <addaction name="ActionTestDelay"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="AcionRegulationSetttings"/>
    <addaction name="ActionUserRuleSettings"/>
    <addaction name="ActionAdditionalSettings"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="ActionGuiLog"/>
    <addaction name="ActionTrojanLog"/>
   </widget>
   <addaction name="menutrojan_go_desktop"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionConnect"/>
   <addaction name="ActionDisconnect"/>
   <addaction name="ActionTestDelay"/>
   <addaction name="ActionEdit"/>
   <addaction name="ActionDel"/>
   <addaction name="ActionShare"/>
   <addaction name="ActionUp"/>
  </widget>
  <action name="ActionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="ActionAdd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="ActionEdit">
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="ActionShare">
   <property name="text">
    <string>分享</string>
   </property>
  </action>
  <action name="ActionDel">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="ActionConnect">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="ActionDisconnect">
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="ActionTestDelay">
   <property name="text">
    <string>测试延迟</string>
   </property>
  </action>
  <action name="AcionRegulationSetttings">
   <property name="text">
    <string>常规设置</string>
   </property>
  </action>
  <action name="ActionUserRuleSettings">
   <property name="text">
    <string>用户规则设置</string>
   </property>
  </action>
  <action name="ActionAdditionalSettings">
   <property name="text">
    <string>高级设置</string>
   </property>
  </action>
  <action name="ActionGuiLog">
   <property name="text">
    <string>gui日志</string>
   </property>
  </action>
  <action name="ActionTrojanLog">
   <property name="text">
    <string>trojan日志</string>
   </property>
  </action>
  <action name="ActionUp">
   <property name="text">
    <string>上移</string>
   </property>
   <property name="toolTip">
    <string>上移</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionList</class>
   <extends>QWidget</extends>
   <header location="global">connectionlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
